File: Tutorials and Problems/P1/MyFirstApp/app/src/main/java/org/feup/apm/myfirstapp/SecondActivity.java
package org.feup.apm.myfirstapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class SecondActivity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    Intent intent = getIntent();
    String message = intent.getStringExtra(FirstActivity.EXTRA_MESSAGE);

    TextView tv_message = new TextView(this);
    tv_message.setTextSize(30f);
    tv_message.setText(message);

    setContentView(tv_message);
  }
}

File: Tutorials and Problems/FirstApp/app/src/main/java/org/feup/apm/firstapp/SecondActivity.kt
package org.feup.apm.firstapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class SecondActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)

    val tv = TextView(this)
    tv.text = intent.getStringExtra(EXTRA_MESSAGE)
    tv.textSize = 30.0f
    setContentView(tv)
  }
}